presto-native-execution/src/test/java/com/facebook/presto/nativeworker/TestPrestoContainerBasicQueries.java

```diff
@@ -0,0 +1,94 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.nativeworker;
+
+import com.facebook.presto.tests.AbstractTestQueryFramework;
+import org.testng.annotations.Test;
+
+import static org.testng.Assert.assertTrue;
+
+public class TestPrestoContainerBasicQueries
+        extends AbstractTestQueryFramework
+{
+    @Override
+    protected ContainerQueryRunner createQueryRunner()
+            throws Exception
+    {
+        return new ContainerQueryRunner();
+    }
+
+    @Test
+    public void testBasics()
+    {
+        assertTrue(
+                computeActual("SELECT * FROM system.runtime.nodes").toString().contains("native-worker-1"),
+                "Native worker is not present.");
+        assertTrue(
+                computeActual("SHOW catalogs").toString().contains("tpch"),
+                "tpch catalog is not present.");
+        assertTrue(
+                computeActual("SHOW session").toString().contains("native_aggregation_spill_all"),
+                "native_aggregation_spill_all is not present.");
+    }
+
+    @Test
+    public void testFunctions()
+    {
+        assertQuery("SELECT COUNT(*) FROM (VALUES 1, 0, 0, 2, 3, 3) as t(x)", "SELECT 6");
+        assertQuery("SELECT array_sort(ARRAY [5, 20, null, 5, 3, 50])", "SELECT ARRAY[3, 5, 5, 20, 50, null]");
+    }
+
+    @Test
+    public void testTables()
+    {
+        assertQuery("SELECT COUNT(*) from orders");
+        assertQuery("SELECT COUNT(*) FROM nation");
+        assertQuery("SELECT COUNT(*) FROM lineitem");
+    }
+
+    @Test
+    public void testUnnest()
+    {
+        assertQuery("SELECT 1 FROM (VALUES (ARRAY[1])) AS t (a) CROSS JOIN UNNEST(a)", "SELECT 1");
```

Comment:
One test per method please, so they can pass, fail, and be debugged independently